### ISSUE TITLE
Freezes at TETRIS TIME on qemu (Windows 10, Ryzen laptop)

### ISSUE THREAD

**Issue by FoxRunTime** [jdah/tetris-os#20]
![image](https://user-images.githubusercontent.com/22801221/115748924-852e0000-a38e-11eb-9f24-149c9229f08c.png)
It just hangs here. 

command used: `qemu-system-i386 -L "C:\Program Files\qemu"  -drive format=raw,file=C:\Users\white\Downloads\boot.iso -d cpu_reset -monitor stdio -device sb16 -audiodev none,id=pulseaudio,out.frequency=48000,out.channels=2,out.format=s32`

**Comment by FoxRunTime** [comment]
update: 
![image](https://user-images.githubusercontent.com/22801221/115749586-25842480-a38f-11eb-8b6d-86caaa11aabf.png)

it does this now

but with garbled sound sometimes.
no video though.

**Comment by farmerbb** [comment]
Try running tetris-os using the method outlined in https://github.com/jdah/tetris-os/issues/2#issuecomment-824773889

**Comment by jdah** [comment]
Yes, please follow what @farmerbb has said. This is also noted in the README, this is a known issue with QEMU running under GTK.

**Comment by TheGamingCoderNoob** [comment]
Same issue but on ArchLinux.

**Comment by Rohanite** [comment]
Whilst this doesn't produce any sound it got it working for me on windows 10: `qemu-system-i386 -display sdl -drive format=raw,file=boot.iso -d cpu_reset -monitor stdio -audiodev sdl,id=sdl,out.frequency=48000,out.channels=2,out.format=s32 -device sb16,audiodev=sdl`

**Comment by zment4** [comment]
> Whilst this doesn't produce any sound it got it working for me on windows 10: `qemu-system-i386 -display sdl -drive format=raw,file=boot.iso -d cpu_reset -monitor stdio -audiodev sdl,id=sdl,out.frequency=48000,out.channels=2,out.format=s32 -device sb16,audiodev=sdl`

Did you try replacing all the audiodev sdl parts with dsound? Works for me when using dsound, not when using sdl for audio. The graphics need to be SDL if you use SB16 as there is a known bug with the GTK renderer and SB16 emulation.

**Comment by jdah** [comment]
There are new suggested instructions for this in the README as it relates to emulating on windows, please check those out.

**Comment by Rohanite** [comment]
> > Whilst this doesn't produce any sound it got it working for me on windows 10: `qemu-system-i386 -display sdl -drive format=raw,file=boot.iso -d cpu_reset -monitor stdio -audiodev sdl,id=sdl,out.frequency=48000,out.channels=2,out.format=s32 -device sb16,audiodev=sdl`
> 
> Did you try replacing all the audiodev sdl parts with dsound? Works for me when using dsound, not when using sdl for audio. The graphics need to be SDL if you use SB16 as there is a known bug with the GTK renderer and SB16 emulation.

`Ah yes, that did work, so the final command was qemu-system-i386 -display sdl -drive format=raw,file=boot.iso -d cpu_reset -monitor stdio -audiodev dsound,id=dsound,out.frequency=48000,out.channels=2,out.format=s32 -device sb16,audiodev=dsound `and that got it working on windows, at least for me
